backend/src/sms/sms.service.ts

```diff
@@ -67,4 +84,12 @@ export class SmsService {
     private getMessage(recipientName: string, systemName: string, link: string, signature: string) {
         return `Hei ${recipientName}! Sinulle on luotu oma Nutakortti. Voit kirjautua palveluun kertakäyttöisen kirjautumislinkin avulla ${link}  - ${signature}`;
     }
+
+    private getExpiredMessage(recipientName: string, expiredDate: string): string {
+        return `Hei\n\nNuoren ${recipientName} Mobiilinutakortti odottaa uusimista kaudelle ${this.getSeasonPeriod()}. Alla olevasta linkistä pääset uusimaan nuoren hakemuksen ja päivittämään yhteystiedot. Edellisen kauden QR-koodi lakkaa toimimasta ${expiredDate}.\n\n${process.env.FRONTEND_BASE_URL ? `${process.env.FRONTEND_BASE_URL}/hae` : 'https://nutakortti.vantaa.fi/hae'}\n\nTerveisin,\nVantaan nuorisopalvelut\n`
```

Comment:
Could you split the message into multiple strings and concatenate them for improved readability?

If we assume that the `expiredDate` is in an ISO string format, it would be good to first parse it and format it using some date library. Luckily we seem to already have `moment.js` as a dependency, so you could say something like
```
const date = moment(expiredDate).format('DD.MM.YYYY');
```
and use that inside the message.